crates/workspace/src/pane_group.rs

```diff
@@ -121,6 +121,35 @@ impl PaneGroup {
         };
     }
 
+    pub fn move_active_item(
```

Comment:
Looking at the code better, it looks like we have 
https://github.com/zed-industries/zed/blob/7b7d5dcd75b729f11ff3764128c772a1140db114/crates/workspace/src/workspace.rs#L6150-L6180

which does almost the same thing but also focuses the right pane?

Let's unity them (by removing this method and moving the existing one here; optionally we can also replace `remove_item_and_focus_on_pane` noop wrapper with `remove_item`).

I think, that focusing the last pane is important, and might solve the observed lag issues.

Reply:
Yep. I missed it when I was reading the code. I'll see where and how it is used and merge them into one if it makes sense.

The lag isn't related to the move. It happens on `ctrl-tab` too. Something to do with chagning the active tab when the tab has a certain LSP enabled, maybe? I'll try to narrow it down

Reply:
I would have used the workspace one instead of the new one, just moved it into the `PaneGroup` (and removed that useless `remove_item_and_focus_on_pane` wrapper).

For the lag, I wonder if it already exists on `main` before the changes?
If focus-related changes do not improve things and it is on `main` already, we can move on and fix that lag separately.